cms/management/commands/create_initial_data.py

```diff
@@ -24,8 +40,20 @@ def collect_initial_data_functions(self):
                         break
         return functions
 
+    def output(self, app_name, verbosity, done=False, result=False):
+        if verbosity > 0:
+            if done:
+                self.stdout.write(self.style.SUCCESS('DONE'))
+            else:
+                self.stdout.write('Creating initial data for {!r}... '.format(app_name), ending='')
+        if verbosity >= 2 and result:
+            pprint.pprint(result)
+
+
     def handle(self, **options):
+        # collect relevant cli arguments
```

Comment:
Style nit: IMO, the following lines are already self-descriptive. We can drop this comment.